shimmingtoolbox/shim/sequencer.py

```diff
@@ -1463,7 +1463,7 @@ def new_bounds_from_currents(currents, old_bounds):
 
     Args:
         currents (np.ndarray): 2D array (n_shims x n_channels). Direct output from :func:`_optimize`.
```

Comment:
Maybe specify, in which class _optimize is ?   ( Sequencer)

Reply:
I removed the comment as it is now a class and did not provide much more information